components/TravelMethodButtons/TravelMethodButtons.jsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Button, Text, SimpleGrid, Icon, Flex } from '@chakra-ui/react';
+
+export function TravelMethodButtons({
+  transportIcon,
+  modesOfTransport,
+  handleTransportMode,
+  transportMode,
+}) {
+  return (
+    <SimpleGrid
+      columns={3}
+      defaultValue={transportMode}
+      id="selector"
+      width={['305px', '548px']}
+      mr={'auto'}
+      ml={'auto'}
+    >
+      {modesOfTransport.map((mode, i) => (
+        <Box height="80px" textAlign={'center'} key={mode}>
+          {/* buttons */}
+
+          <Button
+            variant="outline"
+            fontSize={[13, 15]}
+            color="#044B7F"
+            height={['58px', '57px']}
+            width={['91.67px', '141px']}
+            border="1px"
+            // borderColor="044B7F.300"
+            onChange={handleTransportMode}
+            colorScheme="#044B7F"
+            value={mode}
+          >
+            <Box
+              pos="absolute"
+              top={['3px', '2px']}
+              pb={['4px', '3px']}
+              width={['30px', '32px']}
+              height={['30px', '32px']}
+              mb="12px"
+            >
+              <Icon as={transportIcon[i]} />
+            </Box>
+            {mode}
+          </Button>
```

Comment:
can we extract this Button out into its own component, and call the component `TravelMethodButton` ?
`TravelMethodButton` can be extracted out into its own file (preferably), or left in this file.

Reply:
This will be addressed in a new PR.